solax/inverters/qvolt_hyb_g3_1p.py

```diff
@@ -0,0 +1,164 @@
+            "PV2 Power": (9, Units.W),
+            "Inverter status": (10, Units.NONE, cls.Processors.inverter_modes),
+            "Total inverter yield": (pack_u16(11, 12), Total(Units.KWH), div10),
+            "Today's inverter yield": (13, DailyTotal(Units.KWH), div10),
+            "Battery Voltage": (14, Units.V, div100),
+            "Battery Current": (15, Units.A, twoway_div100),
+            "Battery Power": (16, Units.W, to_signed),
+            "Battery Temperature": (17, Units.C),
+            "Battery Remaining Capacity": (18, Units.PERCENT),
+            "Total battery discharged energy": (
+                pack_u16(19, 20),
+                Total(Units.KWH),
+                div10,
+            ),
+            "Total battery charged energy": (pack_u16(21, 22), Total(Units.KWH), div10),
+            "Battery Remaining Energy": (
+                23,
+                Measurement(Units.KWH, storage=True),
+                div10,
+            ),
+            # 24: always 100. probably 10.0%, minimum charge left before switching to grid
+            # 25: always 0
+            # 26-27: jumping around
+            # 28-31: always 0
+            "Current grid power": (pack_u16(32, 33), Units.W, to_signed32),
+            "Total grid export": (pack_u16(34, 35), Total(Units.KWH), div100),
+            "Total grid import": (pack_u16(36, 37), Total(Units.KWH), div100),
+            "Current power usage": (38, Units.W, to_signed),
+            # 39: observed range 0-38. something about the inverter
+            # 40: 256 when inverter is running, 0 when it's idle
+            # 41-42: fixed values
+            # 43-49: no idea, varying values
+            # 50-51: always 1. possibly an inverter setting
+            # 52: idk, max 4, min 0
+            # 53: always 0
+            "Total self-used solar": (pack_u16(54, 55), Total(Units.KWH), div10),
+            # 56-57: 32 bit pack? mostly negative, went positive briefly
+            # 58-59: 32 bit pack? always negative, between -144 and -175
+            # 60-69: always 0
+            # 70: fluctuates quite wildly
+            # 72-73: 32 bit pack?
+            # 74-77: fixed values
+            "Today's grid export": (pack_u16(78, 79), DailyTotal(Units.KWH), div100),
+            "Today's grid import": (pack_u16(80, 81), DailyTotal(Units.KWH), div100),
+            # 82-84: always 0
+            "Today's solar yield": (85, DailyTotal(Units.KWH), div10),
+            # 86: something daily
+            # 87: something else daily
+            # 88-89: always 0
+            # 90: min 350, max 360
+            # 91: always 0
+            # 92: always 230
+            # 93-95: something daily
+            # 97-98: always 0
+            # 99-100: weird counter. rtc?
+            # 101-109: fixed values
+            # 110: battery type!?
+            # 111-115: mysterious stuff
+            "Total battery energy throughput": (pack_u16(116, 117), Total(Units.WH)),
```

Comment:
Use `Measurement(Units.X, storage=True)` instead of `Total(Units.X)`.

You could consider adding `div1000` to `solax.utils` and have the unit be `Units.KWH`.